includes/services/extended/0-google-base.php

```diff
@@ -253,6 +253,9 @@ function basic_ui() {
 		echo '</p>';
 		echo '</form>';
 		echo '</div>';
+		if($this->supports_basic_auth){
```

Comment:
needed that due to override of the `basic_ui()` function in the google-base service